ext/native-decls/SetMpGamerTagsUseVehicleBehavior.md

```diff
@@ -1,14 +1,15 @@
 ---
 ns: CFX
 apiset: client
+aliases: ["0x6DD05E9D83EFA4C9","_SET_MP_GAMER_TAGS_USE_VEHICLE_BEHAVIOR"]
```

Comment:
This second alias you added `_SET_MP_GAMER_TAGS_USE_VEHICLE_BEHAVIOR` will never work since it won't match the one registered by the native handler. 

Also not quite sure where you got that first alias from `0x6DD05E9D83EFA4C9`, looks like you got that from `CREATE_MP_GAMER_TAG_WITH_CREW_COLOR`.